src/mongocxx/test/client.cpp

```diff
@@ -15,6 +15,10 @@
 #include "catch.hpp"
 #include "helpers.hpp"
 
+#define MONGOC_I_AM_A_DRIVER true
```

Comment:
Lets not worry about this just yet. I'm fine with doing this for now. We can always address later if it is an issue.